docs/esp32/quickref.rst

```diff
@@ -365,6 +365,22 @@ Notes:
     
     p1 = Pin(4, Pin.OUT, None)
 
+RMT
+---
+
+The RMT is a module, specific to the ESP32, that allows pulses of very accurate interval
+and duration to be sent and received::
```

Comment:
I'd expand this paragraph to show RMT is short for Remote Control and that it was originally built for IR but can be repurposed.

```
The RMT (Remote Control) module, specific to the ESP32, was designed to send and receive infrared remote control signals. Due to flexibility of module, the driver can also be used to generate or receive many other types of digital signals. The module allows pulses of very accurate interval and duration to be sent and received::
```

Reply:
Good suggestion! I've updated the paragraph accordingly.